addons/l10n_jo/data/template/account.tax-jo_standard.csv

```diff
@@ -1,77 +1,77 @@
 "id","name","type_tax_use","amount","amount_type","description","invoice_label","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","name@ar_001"
-"jo_standard_purchase_16","16%","purchase","16.0","percent","16% (2a)","16%","jo_tax_group_16","base","invoice","+2a (Base)","",""
+"jo_standard_purchase_16","16%","purchase","16.0","percent","","16%","jo_tax_group_16","base","invoice","+2a (Base)","",""
 "","","","","","","","","tax","invoice","+2a (Tax)","jo_account_200901",""
 "","","","","","","","","base","refund","-2a (Base)","",""
 "","","","","","","","","tax","refund","-2a (Tax)","jo_account_200901",""
-"jo_standard_purchase_10","10%","purchase","10.0","percent","10% (2b)","10%","jo_tax_group_10","base","invoice","+2b (Base)","",""
+"jo_standard_purchase_10","10%","purchase","10.0","percent","","10%","jo_tax_group_10","base","invoice","+2b (Base)","",""
 "","","","","","","","","tax","invoice","+2b (Tax)","jo_account_200901",""
 "","","","","","","","","base","refund","-2b (Base)","",""
 "","","","","","","","","tax","refund","-2b (Tax)","jo_account_200901",""
-"jo_standard_purchase_4","4%","purchase","4.0","percent","4% (2c)","4%","jo_tax_group_4","base","invoice","+2c (Base)","",""
+"jo_standard_purchase_4","4%","purchase","4.0","percent","","4%","jo_tax_group_4","base","invoice","+2c (Base)","",""
 "","","","","","","","","tax","invoice","+2c (Tax)","jo_account_200901",""
 "","","","","","","","","base","refund","-2c (Base)","",""
 "","","","","","","","","tax","refund","-2c (Tax)","jo_account_200901",""
-"jo_standard_purchase_other","0% O","purchase","0.0","percent","0% (2d) Other Tax","0%","jo_tax_group_0","base","invoice","+2d (Base)","","2d. ضريبة المشتريات الأخرى"
+"jo_standard_purchase_other","0% O","purchase","0.0","percent","Other","0% Other","jo_tax_group_0","base","invoice","+2d (Base)","","2d. ضريبة المشتريات الأخرى"
```

Comment:
i know it's not you but the translation seems a bit weird, could we have confirmation on that ? also maybe we need to translation the description then

Reply:
Yeah seems like the translation were already meant to be for the descriptions. I fixed it now in a separate commit, should I merge the 2 commits into 1 or what do u think?

Reply:
One commit is ok i think :smile:

Reply:
Also, wouldn't removing names translations from stable be an issue?

Reply:
For csv like here i think it's ok :smile: